scripts/Frequency_Domain_Analysis.py

```diff
@@ -1,14 +1,15 @@
 #pylint: disable=invalid-name
 from __future__ import (absolute_import, division, print_function)
+import PyQt4.QtGui as QtGui
```

Comment:
I'm only commenting here because I've got a few other comments but the order should be

```
import sys

import PyQt4.QtGui as QtGui

from Muon import FFT_presenter
from Muon import FFT_view
```